### ISSUE TITLE
Update Sample and Experiment models with organism information

### ISSUE THREAD

**Issue by mhuyck** [greenelab/adage-server#165]
In reviewing PR #161, @cgreene notes on `interface/src/app/analyze/analysis/analysisDetail.tpl.html`:

> I'm imagining a situation where a user searches and selects datasets from multiple organisms. How will we show those results? I can't think of a sensible way to do that. Restricting by mlmodel first prevents us from getting stuck in a situation where the user is selecting multiple experiments that we can't sensibly show together (if we facet the search results by the mlmodel's organism.

At the moment, there is no link between Experiment or Sample to an organism. To enforce a consistent organism among the samples used for an analysis we will need to update the models and add features to the search to assist users.

**Comment by cgreene** [comment]
I think an Experiment is M2M with organism. @kurtwheeler can help us snag the details. But for now, I think assuming a sample is a one-to-one mapping with organisms is a safe bet. There will be some other cases that exist, but I do not think it is a good use of our time to deal with those at this time.

**Comment by kurtwheeler** [comment]
I would agree both that experiment should be M2M with organism (I don't know if I've necessarily seen an example of this but it seems impossible to rule out) and that assuming a sample is a one-to-one mapping with organisms is a safe bet.

The Data Refinery links each batch to an organism. For Array Express this means that each sample is linked to an organism. This is actually a good consideration for the future if we ever are considering making using an experiment as a batch for a different source.
